lib/fca/row.rb

```diff
@@ -3,16 +3,29 @@ class Row
     HEADER_NAME = 1
     REFERENCE_NUMBER = 0
     NAME = 1
-    ADVISER_STATUS_CODE = 4
-    FIRM_AUTHORISATION_STATUS_CODE = 19
-    SUBSIDIARY_END_DATE = 4
 
-    ACTIVE_FIRM_AUTHORISATION_STATUS_CODES = [
+    #ADVISER_STATUS_CODE = 4
+    #FIRM_AUTHORISATION_STATUS_CODE = 19
+    #SUBSIDIARY_END_DATE = 4
+
+    #ACTIVE_FIRM_AUTHORISATION_STATUS_CODES = [
+    #  'Authorised',
+    #  'Registered',
+    #  'EEA Authorised'
+    #].freeze
+
+    ADVISER_STATUS_INDEX = 2
```

Comment:
You've added `INDEX` suffixes and that makes sense, but that doesn't follow the existing style for the other constant column indices here. Could make sense to either match the existing style or add another commit, making the existing ones consistently named with the `INDEX` suffix?